lib/mrb/mrb_logger.c

```diff
@@ -57,6 +57,18 @@ logger_s_get_default_level(mrb_state *mrb, mrb_value self)
   return mrb_funcall(mrb, mrb_level_class, "find", 1, mrb_level);
 }
 
+static mrb_value
+logger_s_get_default_flags(mrb_state *mrb, mrb_value self)
+{
+  mrb_value mrb_flags_class;
+  mrb_value mrb_flags;
+
+  mrb_flags_class = mrb_const_get(mrb, self, mrb_intern_lit(mrb, "Flag"));
+  mrb_flags = mrb_fixnum_value(grn_default_logger_get_flags());
+  return mrb_funcall(mrb, mrb_flags_class, "set_flags", 1, mrb_flags);
+}
+
+
```

Comment:
Could you remove an empty needless line?

Reply:
fixed.